test/parallel/test-readline-interface.js

```diff
@@ -71,6 +71,19 @@ function assertCursorRowsAndCols(rli, rows, cols) {
   assert(rl instanceof readline.Interface);
 }
 
+{
+  const fi = new FakeInput();
+  const rli = new readline.Interface(
+    fi,
+    fi,
+    common.mustCallAtLeast((line) => [[], line]),
```

Comment:
Does `common.mustCall()` work here, or does it need to be `common.mustCallAtLeast()`? If `common.mustCall()` works, I think that is a slightly better option.

Reply:
Thank you. `common.mustCall()` works here as you suggested. I will use `common.mustCall()`.